packages/tools/workloads-manager/package.json

```diff
@@ -18,6 +18,7 @@
     "build": "rollup -c --bundleConfigAsCjs",
     "build:apps": "TYPE='static' node scripts/build.js",
     "connect": "DATA='./workloads.config.json' node scripts/connect.js",
+    "move": "DATA='./workloads.config.folder.json' node scripts/move.js",
```

Comment:
General side-note: I'm not a too big fan of passing arguments via ENV vars, would be a lot of work to get that working via command line options?

e.g. something like `node scripts/move.js --data='./workloads.config.folder.json'`

Out of experience this is less error prone and easier to debug .

Reply:
Great point - that should be done in a separate pr, since I use them and I'm a fan 🤣 

Since we're still testing locally and manually, I'll go ahead and merge this pr.